server/broadcast.go

```diff
@@ -114,8 +115,8 @@ func (bsm *BroadcastSessionsManager) completeSession(sess *BroadcastSession) {
 
 func (bsm *BroadcastSessionsManager) refreshSessions() {
 
-	glog.V(common.DEBUG).Info("Starting session refresh")
-	defer glog.V(common.DEBUG).Info("Ending session refresh")
+	glog.V(common.DEBUG).Info("Starting session refresh manifestID=%v", bsm.mid)
```

Comment:
This needs to be `.Infof` , thanks Circle !

Reply:
Alternatively, `Info("Starting session refresh manifestID=", bsm.mid)` would work (without the %v format specifier)

Reply:
Fixed in 8a26ba9